jbs-ubi8-builder/Dockerfile

```diff
@@ -44,9 +44,9 @@ RUN set -o errexit -o nounset \
     \
     \
     && echo "Downloading Ant 1.10.15" \
-    && wget --no-verbose --output-document=ant.zip "https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.15-bin.zip" \
+    && wget --no-verbose --output-document=ant.zip "https://dlcdn.apache.org/ant/binaries/apache-ant-1.10.15-bin.zip" \
     && echo "Checking download hash 1.10.15" \
-    && echo "1de7facbc9874fa4e5a2f045d5c659f64e0b89318c1dbc8acc6aae4595c4ffaf90a7b1ffb57f958dd08d6e086d3fff07aa90e50c77342a0aa5c9b4c36bff03a9 *ant.zip" | sha512sum --check - \
+    && echo "e59baf898dc5b6d1aa6cd57544715f7e0060b1fdc9e56f24f9898d56612a9e0b *ant.zip" | sha256sum --check - \
```

Comment:
Was there a reason to use the SHA256 sum? Where did that come from? Given the SHA512 is on https://downloads.apache.org/ant/binaries/apache-ant-1.10.15-bin.zip.sha512 ?

Reply:
actually no, I saw where I should change the template to use 512 as before. thanks for noticing!